src/test/java/org/example/AddRemoveItem.java

```diff
@@ -1,7 +1,109 @@
 package org.example;
 
 
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
 
 public class AddRemoveItem {
 
+    private WebDriver webDriver;
+
+    @BeforeMethod
+    public void setUpDrive() {
+        System.setProperty(
+                "webdriver.chrome.driver",
+                "src/test/resources/webdriver/chromedriver.exe"
+        );
+
+        webDriver = new ChromeDriver();
+    }
+
+    @Test
+    public void firstTest() {
+
+        webDriver.get("https://www.amazon.com/");
+        webDriver.manage().window().maximize();
+
+        WebElement category = webDriver.findElement(By.xpath("//a[@aria-label=\"Computers & Accessories\"]"));
+        category.click();
+
+
+        WebElement openItem = webDriver.findElement(By.xpath("//*[contains(text(),'Seagate Portable')]"));
+        openItem.click();
+
+
+        WebElement addButton = webDriver.findElement(By.id("add-to-cart-button"));
+        addButton.click();
+
+
+        WebElement addedToCart = webDriver.findElement(By.xpath("//*[@id=\"NATC_SMART_WAGON_CONF_MSG_SUCCESS\"]/span"));
+        Assert.assertEquals("Added to Cart", addedToCart.getText());
+
+        WebElement cartNumber = webDriver.findElement(By.xpath("//*[@id=\"nav-cart-count\"]"));
+        Assert.assertEquals("1", cartNumber.getText());
+
+
+    }
+
+
+    @Test
+    public void secondTest() {
+
+        webDriver.get("https://www.amazon.com/");
+        webDriver.manage().window().maximize();
+
+        WebElement category = webDriver.findElement(By.xpath("//a[@aria-label=\"Computers & Accessories\"]"));
+        category.click();
+
+
+        WebElement openItem = webDriver.findElement(By.xpath("//*[contains(text(),'Seagate Portable')]"));
+        openItem.click();
+
+
+        WebElement addButton = webDriver.findElement(By.id("add-to-cart-button"));
+        addButton.click();
+
+
+        WebElement addedToCart = webDriver.findElement(By.xpath("//*[@id=\"NATC_SMART_WAGON_CONF_MSG_SUCCESS\"]/span"));
+        Assert.assertEquals("Added to Cart", addedToCart.getText());
+
+        WebElement cartNumber = webDriver.findElement(By.xpath("//*[@id=\"nav-cart-count\"]"));
+        Assert.assertEquals("1", cartNumber.getText());
+
+        //-----
```

Comment:
What is the reason for the comment

Reply:
Until this comment, the code is the same as in the first test. I put this comment to show where the extra code begins.

Reply:
i think is not necessary

Reply:
Removed the comment